src/main.cpp

```diff
@@ -2542,9 +2547,9 @@ bool CheckBlock(const CBlock& block, CValidationState& state, bool fCheckPOW, bo
 
 bool ContextualCheckBlockHeader(const CBlockHeader& block, CValidationState& state, CBlockIndex * const pindexPrev)
 {
-    const Consensus::Params& consensusParams = Params().GetConsensus();
```

Comment:
Oh, now I get the warning you're complaining about, which has nothing to do with this PR.
https://github.com/bitcoin/bitcoin/blob/master/src/main.cpp#L4844 was introduced in https://github.com/theuni/bitcoin/commit/e8e8904d#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR4503 and should have been removed when ```consensusParams.nPowTargetSpacing``` was not used in the function anymore.
Not sure who's fault is, but probably whoever did it didn't noticed while rebasing.
Something trivial to fix though.